.circleci/config.yml

```diff
@@ -148,6 +148,18 @@ installdetectrondeps: &installdetectrondeps
       cd maskbench; git checkout 4c168a637f45dc69efed384c00a7f916f57b25b8 -b stable
       python setup.py develop; cd -
 
+installcrowdsourcingdeps: &installcrowdsourcingdeps
+  run:
+    name: Install Mephisto
+    command: |
+      cd ..
+      git clone git@github.com:facebookresearch/Mephisto.git Mephisto
+      cd Mephisto; git checkout a9abd9deb2d2a5e0d6fde7dd6873fb73b4cb4872 -b stable
```

Comment:
I suppose there's going to need to be some stable version of Mephisto that ParlAI is set to work with? Not sure the best way to get that in here.

Reply:
hmm, if Mephisto supports `pip install` and version numbering in the future, I could just add a row for it in `requirements.txt` :)